src/Config.cpp

```diff
@@ -8,7 +8,7 @@ See LICENSE for licensing.
 
 using namespace std;
 
-string SQUIDversion="v1.4";
+string SQUIDversion="v1.5";
```

Comment:
Did you change the squid version number? Not sure we should do that if we upstream changes, or have a different versioning scheme if we don't upstream.